presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java

```diff
@@ -161,6 +161,19 @@ public static Slice absLong(@SqlType("decimal(p, s)") Slice arg)
         }
     }
 
+    @ScalarFunction("log")
+    @Description("logarithm to given base")
+    public static final class LogFunction
```

Comment:
This no longer needs to be a class -- it can just be a function. Let's move it to be next to the other log functions.